docs/reference/query-languages/esql.md

```diff
@@ -1,33 +1,23 @@
 ---
-navigation_title: "{{esql}} reference"
+navigation_title: "{{esql}}"
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-language.html
 ---
 
 # {{esql}} reference [esql-language]
 
 :::{note}
-This section provides detailed **reference information**.
+This section provides detailed **reference information** about the {{esql}} language, including syntax, functions, and operators.
 
-For overview, conceptual, and getting started information, refer to the [{{esql}} language](docs-content://explore-analyze/query-filter/languages/esql.md) in the **Explore and analyze** section.
+For overview, conceptual, and getting started information, refer to the [{{esql}} language overview](docs-content://explore-analyze/query-filter/languages/esql.md) in the **Explore and analyze** section.
 :::
 
-* [Syntax reference](/reference/query-languages/esql/esql-syntax.md)
-* [Commands](/reference/query-languages/esql/esql-commands.md)
-* [Functions overview](/reference/query-languages/esql/esql-functions-operators.md#esql-functions)
-* [Metadata fields](/reference/query-languages/esql/esql-metadata-fields.md)
-* [Multivalued fields](/reference/query-languages/esql/esql-multivalued-fields.md)
-* [Data enrichment](/reference/query-languages/esql/esql-enrich-data.md)
-* [Data processing with DISSECT and GROK](/reference/query-languages/esql/esql-process-data-with-dissect-grok.md)
-* [Implicit casting](/reference/query-languages/esql/esql-implicit-casting.md)
-* [Time spans](/reference/query-languages/esql/esql-time-spans.md)
-
-
-
-
-
-
-
-
+{{esql}} is a SQL-like query language for querying and analyzing time series data in {{es}}. It is designed to be easy to use and understand, while also being powerful enough to handle complex queries.
```

Comment:
ESQL is not really SQL-like (except for the rare inspiration we draw from good ol' SQL), and it's not limited to time series data. Maybe like this?
```suggestion
{{esql}} is a piped query language for exploring and analyzing data in {{es}}. It is designed to be easy to use and understand, while also being powerful enough to handle complex data processing.
```